openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml

```diff
@@ -556,28 +556,45 @@
 - control_key: IA-5 (2)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
   narrative:
     - key: a
       text: |-
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-327
+        The certificates used for in-cluster mTLS authentication
+        are issued by an internal CA, at the moment it is not possible
+        to set a different CA:
+        https://docs.openshift.com/container-platform/4.6/security/certificate_types_descriptions/control-plane-certificates.html
+
+        The API server's certificates can be swapped for user-provided
+        ones:
+        https://docs.openshift.com/container-platform/4.6/security/certificate_types_descriptions/user-provided-certificates-for-api-server.html
+
+        It is also possible to set a custom CA for the Proxy objects:
+        https://docs.openshift.com/container-platform/4.6/security/certificate_types_descriptions/proxy-certificates.html
+        services:
+        https://docs.openshift.com/container-platform/4.6/security/certificate_types_descriptions/service-ca-certificates.html
+
+        When setting up IDPs, all compliant IDP allow the CA that signed
+        the IDP certificate to be set:
+        https://docs.openshift.com/container-platform/4.7/rest_api/config_apis/oauth-config-openshift-io-v1.html
     - key: b
       text: |-
-        A control response is planned. Engineering progress can be tracked via:
+        Most of the private keys are stored in Kubernetes secrets
```

Comment:
nit: Most would imply there are still some that are not protected, so we might need to find and mention the exceptions.

Reply:
I didn't list the exceptions one-by-one, but added a more generic blob based on CIS 1.1.21, what do you think?